NuRadioReco/detector/detector.py

```diff
@@ -156,6 +152,8 @@ def __init__(self, source='json', json_filename='ARIANNA/arianna_detector_db.jso
             Can be used to force the creation of a new detector object. By default, the __init__ will anly create a new
             object of none already exists.
         """
+        self._serialization = SerializationMiddleware()
```

Comment:
Do you want a `if self._serialization = None` here? Otherwise I don't see the point in passing the serialization as an argument?